ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -30,7 +30,7 @@
     private static final By START_VISIT = By.id("org.openmrs.module.coreapps.createVisit");
     private static final By CONFIRM = By.cssSelector("#quick-visit-creation-dialog .confirm");
     private static final By STARTED_AT = By.className("active-visit-started-at-message");
-    private static final By VISIT_NOTE = By.id("referenceapplication.realTime.simpleVisitNote");
```

Comment:
The element can uniquely be retrieved by _linkText()_ attribute. Other attribute properties _(id, cssSelector, xpath)_  fail to retrieve the element